manifest/v1alpha/direct/validation.go

```diff
@@ -24,7 +24,7 @@ var directValidation = validation.New[Direct](
 var specValidation = validation.New[Spec](
 	validation.For(validation.GetSelf[Spec]()).
 		Rules(exactlyOneDataSourceTypeValidationRule).
-		StopOnError().
+		Cascade(validation.CascadeModeStop).
```

Comment:
Shouldn't that be placed before `Rules(exactlyOneDataSourceTypeValidationRule).`

Reply:
yes :) thanks